codalab/lib/codalab_manager.py

```diff
@@ -497,6 +552,69 @@ def client(self, address, is_cli=True):
     def cli_verbose(self):
         return self.config.get('cli', {}).get('verbose')
 
+    def get_access_token(self, address, auth_handler):
```

Comment:
I did the refactoring for you: https://github.com/codalab/codalab-cli/pull/454

I really don't like unnecessary copied code. It might be some time until the cl server is completely removed and until then there's this giant block of code that's duplicated.